aquarium.py

```diff
@@ -69,7 +69,7 @@ def build_src(src_dir, version, git_url):
         subprocess.check_call(["git", "-C", f"{SRC_DIR}", "clone", git_url])
 
     subprocess.check_call(["git", "-C", f"{src_dir}", "fetch", "origin", version])
```

Comment:
To support fetching tags as well:
```suggestion
    subprocess.check_call(["git", "-C", f"{src_dir}", "fetch", "origin"])
```